src/EventListener/WriteListener.php

```diff
@@ -25,10 +26,12 @@
 class WriteListener
 {
     private $dataPersister;
+    private $iriConverter;
 
-    public function __construct(DataPersisterInterface $dataPersister)
+    public function __construct(DataPersisterInterface $dataPersister, IriConverterInterface $iriConverter)
```

Comment:
this is a BC break

Reply:
The second argument must be made optional. :smile:

But we can trigger a deprecation error if it's not provided.

Reply:
haha sorry yeah I should've say what you said in addition :laughing: